service/application/src/main/java/com/solace/maas/ep/common/messages/ScanDataImportMessage.java

```diff
@@ -16,8 +16,6 @@ public class ScanDataImportMessage extends MOPMessage {
 
     String scanId;
 
-    String traceId;
```

Comment:
The MOPMessage class already has a traceId field so no need to introduce a new field (which eclipses the existing one).

This is done for all of the message type defintions.